Maddori.Apple/Maddori.Apple/Screen/MyReflection/FeedbackToMeDetail/FeedbackToMeDetailViewController.swift

```diff
@@ -0,0 +1,187 @@
+        let label = UILabel()
+        label.setTitleFont(text: model.keyword)
+        label.textColor = .black100
+        return label
+    }()
+    private let feedbackTypeLabel: UILabel = {
+        let label = UILabel()
+        label.text = TextLiteral.feedbackToMeDetailViewControllerFeedbackTypeLabel
+        label.textColor = .black100
+        label.font = .label2
+        return label
+    }()
+    private lazy var feedbackTypeText: UILabel = {
+        let label = UILabel()
+        label.setTextWithLineHeight(text: model.feedbackType.rawValue, lineHeight: 24)
+        label.textColor = .gray400
+        label.font = .body1
+        return label
+    }()
+    private let feedbackFromLabel: UILabel = {
+        let label = UILabel()
+        label.text = TextLiteral.feedbackToMeDetailViewControllerFeedbackFromLabel
+        label.textColor = .black100
+        label.font = .label2
+        return label
+    }()
+    private lazy var feedbackFromText: UILabel = {
+        let label = UILabel()
+        label.setTextWithLineHeight(text: model.from, lineHeight: 24)
+        label.textColor = .gray400
+        label.font = .body1
+        return label
+    }()
+    private let feedbackContentLabel: UILabel = {
+        let label = UILabel()
+        label.text = TextLiteral.feedbackToMeDetailViewControllerFeedbackContentLabel
+        label.textColor = .black100
+        label.font = .label2
+        return label
+    }()
+    private lazy var feedbackContentText: UILabel = {
+        let label = UILabel()
+        label.setTextWithLineHeight(text: model.content, lineHeight: 24)
+        label.textColor = .gray400
+        label.font = .body1
+        label.numberOfLines = 0
+        return label
+    }()
+    private let feedbackStartLabel: UILabel = {
+        let label = UILabel()
+        label.text = TextLiteral.feedbackToMeDetailViewControllerFeedbackStartLabel
+        label.textColor = .black100
+        label.font = .label2
+        return label
+    }()
+    private lazy var feedbackStartText: UILabel = {
+        let label = UILabel()
+        label.setTextWithLineHeight(text: model.start, lineHeight: 24)
+        label.textColor = .gray400
+        label.font = .body1
+        label.numberOfLines = 0
+        return label
+    }()
+    
+    // MARK: - life cycle
+    
+    override func configUI() {
+        super.configUI()
+        setupStartLabel()
+    }
+    
+    override func render() {
+        view.addSubview(myReflectionScrollView)
+        myReflectionScrollView.snp.makeConstraints {
+            $0.edges.equalToSuperview()
+        }
+        
+        myReflectionScrollView.addSubview(myReflectionContentView)
+        myReflectionContentView.snp.makeConstraints {
+            $0.edges.equalTo(myReflectionScrollView.snp.edges)
+            $0.width.equalTo(myReflectionScrollView.snp.width)
+            $0.height.equalTo(myReflectionScrollView.snp.height).offset(50)
+        }
```

Comment:
p1;

```suggestion
        myReflectionContentView.snp.makeConstraints {
            $0.width.top.bottom.equalToSuperview()
        }
```

현재 내부 컨텐츠가 매우 길어졌을때 스크롤이 애매하게 걸리는 상황이 있습니다 !
width 와 top bottom을 픽스한뒤, 제일 마지막 요소의 bottom값을 superview와 같게 설정해주세요 !

https://user-images.githubusercontent.com/78677571/201792693-b95ddda2-ed35-4e38-84f7-49c20121a2c6.mp4

Reply:
최고의 선생님 !!!! ⭐️
[6175961](https://github.com/DeveloperAcademy-POSTECH/MacC-Team-Maddori.Apple/pull/98/commits/61759614eac1aa77e23f2cde5b3f968214c4aa3c)